backend/package.json

```diff
@@ -24,7 +24,8 @@
     "dotenv": "^16.0.3",
     "express": "^4.21.2",
     "form-data": "^4.0.1",
-    "opentok": "^2.16.0"
```

Comment:
Just moving things around to make sure all dependencies required to build and run the project are in dependencies not devDependencies. By default VCR will run yarn in production mode, only installing dependencies.